### PR TITLE
Update imageIDI.py

### PR DESCRIPTION
**Pull request by dentalfloss1**
Implement fits basename. Does the same as PR #67, but into main

### CODE REVIEW

File: scripts/imageIDI.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def main(args):
                 if yn not in ('n', 'N'):
                     overwrite = True
             try:
-                hdulist.writeto(args.fits, overwrite=overwrite)
+                hdulist.writeto(f"{args.fits}-t{set}.fits", overwrite=overwrite)
```
Comment (jaycedowell) [inline scripts/imageIDI.py:238]: 1. There's a line above this where it checks for the existence of a file called `args.fits` that needs to be updated as well.
 2. I haven't officially dropped support for Py2 so we can't assume f-string support.

I'll take care of both of these.

**Review by jaycedowell — CHANGES_REQUESTED** [review]

**Comment by jaycedowell** [comment]
Do these changes work for you?

**Comment by dentalfloss1** [comment]
It looks good to me. I can't properly test it since I need the lwana branch to process the files I am working with.

**Review by jaycedowell — APPROVED** [review]

**Comment by jaycedowell** [comment]
It's also been merged into lwana.

**Pull request merged**